handbook/data-stores/mongodb.md

```diff
@@ -0,0 +1,133 @@
+|<img src="../../img/mdb-atlas.png" width="1500px"/>| Multi node with MongoDB Atlas | You have SystemLink hosted on a dedicated server and connect to a MongoDB Atlas cluster. Use for large scale deployments (>200 managed nodes, 10,000 test steps and results daily). MongoDB Atlas also simplifies database provisioning, operation, backup and restore. These features may be desireable at any scale of deployment. Refer to [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) for details on this service.
+
+## Single Node Deployments
+
+By default, SystemLink stores all data on locally hosted databases and the local file system. For more reliable data storage and scalability, NI recommends [remote data stores](#multi-node-deployments) for all production deployments.
+
+### Increasing the index cache size for *single node* deployments
+
+Increase the index cache size to process more data and use the Test Insights or Asset Management modules in a single-node deployment. This avoids high CPU usage that can occur with the default MongoDB memory constraint in SystemLink. NI recommends increasing the available memory for the index to 8GB. Refer to [Sizing a SystemLink Server](https://www.ni.com/documentation/en/systemlink/latest/setup/sizing-a-systemlink-server/) for system requirements to run SystemLink.
+  
+1. Log into the desktop of the SystemLink app server with a user with administrator privileges.
+1. Open **NI SystemLink Server Configuration**.
+1. Navigate to **NoSqlDatabase**.
+1. Under **The NoSqlServer is launched and managed by SystemLink**, adjust the value in **The cache size of the database in Gigabytes (GB)**.
+1. Click **Apply**.
+
+<figure>
+  <img src="../../img/mdb-cache.png" width="500" />
+  <figcaption>SystemLink's NoSqlDatabase configuration set to 8BG</figcaption>
+</figure>
+
+## Multi Node Deployments
+
+SystemLink supports three types of multi-node configuration. You can configure SystemLink to connect to a standalone instance, replica sets, or MongoDB Atlas.
+
+NI recommends connecting SystemLink to replica set of three or more MongoDB members or MongoDB Atlas for all production deployments.
+
+- NI does not support configurations where multiple SystemLink servers use the same MongoDB instance.
+- NI does not support configurations where another production application uses the same MongoDB instance as SystemLink.
+- While SystemLink can connect to and use a sharded MongoDB cluster (`mongos`), it will not take advantage of horizontal scaling capabilities enabled by sharded clusters.
+- When using MongoDB Atlas or [MongoDB Enterprise Advanced](https://www.mongodb.com/products/mongodb-enterprise-advanced), you have the option to encrypt the data stored within the database.
+
+!!!important "Connection String Formats"
+    Some connection string formats are unsupported or could cause issues in some environments.
```

Comment:
```suggestion
    The following connection string formats are unsupported or could cause issues in some environments:
```

Reply:
Would it be better to label this section as known issues with connection strings?